src/sql/src/parser.rs

```diff
@@ -217,6 +223,36 @@ impl<'a> ParserContext<'a> {
         Ok(Statement::ShowTables(ShowTables { kind, database }))
     }
 
+    /// Parses DESCRIBE statements
+    fn parse_describe(&mut self) -> Result<Statement> {
+        if self.matches_keyword(Keyword::TABLE) {
+            self.parser.next_token();
+            self.parse_describe_table()
+        } else {
+            self.unsupported(self.peek_token_as_string())
+        }
+    }
+
+    fn parse_describe_table(&mut self) -> Result<Statement> {
```

Comment:
`parse_describe_table` should also parse catalog name and schema name. You may need [`table_idents_to_full_name`](https://github.com/GreptimeTeam/greptimedb/blob/872ac8058f89a22d82fc8df785494901531f101e/src/sql/src/statements.rs#L44) function to convert a `ObjectName` to full-qualified table name with catalog and schema.